### PR TITLE
WIP schemas: fix orcid pattern

### PR DESCRIPTION
**Pull request by jacquerie**
Baby step towards solving https://sentry.cern.ch/inspire-sentry/inspire-labs-qa/group/601553/.

Closes #1407

### CODE REVIEW

File: inspirehep/modules/records/jsonschemas/records/hep.json [hunk 1]
```diff
@@ -195,10 +195,8 @@
                         "type": "string"
                     },
                     "orcid": {
-                        "description": "ORCID Id when available",
                         "format": "orcid",
-                        "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{4}",
-                        "title": "ORCID ID",
```
Comment (david-caro) [inline inspirehep/modules/records/jsonschemas/records/hep.json]: Why removing the other properties? (seems unrelated to the untrained eye, aka, me :wink: )

Comment (jacquerie) [inline inspirehep/modules/records/jsonschemas/records/hep.json]: Mh, I've come to despise `title` and `description` that add nothing to the name of the field. But yeah, they are unrelated, so I will retract these changes.

Comment (david-caro) [inline inspirehep/modules/records/jsonschemas/records/hep.json]: As long as you know what you are doing I'm ok with it, just making sure it's not an 'unwanted' change. Maybe add a small note on the commit message.

**Comment by jacquerie** [comment]
Will be merged in #1392.
